src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml.cs

```diff
@@ -390,7 +394,6 @@ void OnWorkspaceViewDataContextChanged(object sender, DependencyPropertyChangedE
                 ViewModel.RequestAddViewToOuterCanvas += vm_RequestAddViewToOuterCanvas;
                 ViewModel.WorkspacePropertyEditRequested += VmOnWorkspacePropertyEditRequested;
                 ViewModel.RequestSelectionBoxUpdate += VmOnRequestSelectionBoxUpdate;
-                ViewModel.RequestNodeAutoCompleteSearch += ShowHideNodeAutoCompleteControl;
```

Comment:
I remember this was added so that Node AutoComplete works for custom node workspace. Any reason of removing this? If intended, would you test using node autocomplete in a new custom node?

Reply:
I see. This was causing the `Show` function to be called two times in a row. I can add it back to avoid affecting custom nodes, but it does seem suspicious.

Reply:
Instead of removing this handler assignment I removed the other one. Tested in both standard and custom node workspace  and it's working. This also prevents the existing behaviour of the handler being called twice on standard workspaces.

Reply:
Thank you @mmisol . Looks reasonable,